.gitignore

```diff
@@ -1,3 +1,4 @@
+.DS_store
```

Comment:
Can you add a comment above this? And an extra line after.

Reply:
updated